daemon/api_prompting.go

```diff
@@ -100,6 +100,13 @@ func getUserID(r *http.Request) (uint32, Response) {
 	return uint32(userIDInt), nil
 }
 
+// isClientActivity returns true if the request comes a prompting handler
+// service.
+func isClientActivity(c *Command, r *http.Request) bool {
```

Comment:
XXX: should this return an error in addition to the boolean? I don't know what a real implementation will look like later but without deeper context I think it might be.

Reply:
I don't think want to return an HTTP error even if we for some reason can't determine whether a request is from a `handler-service` client, and I'd just as soon wrap that graceful error handling logic in the `isClientActivity` function anyway, so callers just worry about the boolean.

I also think the whole `isClientActivity` helper will be rewritten once we know how we want to implement this (and parameters/returns may change) so I'd just as soon put off handling of errors we can't yet create until we know what those errors will look like :)

Reply:
I also created a task to make sure we get back to this: https://warthogs.atlassian.net/browse/SNAPDENG-34251